FreeDesktop/openfodder.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Open Fodder
+GenericName=Action-Strategy Shoot 'em 
+Comment=An open source port of Cannon Fodder
+Icon=openfodder
+Exec=openfodder
```

Comment:
lowercase is only used by CMake (since #36):
https://github.com/OpenFodder/openfodder/blob/8504e7ab7d44d435539a2910bb63a0e2af7d20b1/CMakeLists.txt#L43

while plain Make and data folder (with game resources) use CamelCase:
https://github.com/OpenFodder/openfodder/blob/8504e7ab7d44d435539a2910bb63a0e2af7d20b1/Makefile#L13
https://github.com/OpenFodder/openfodder/blob/8504e7ab7d44d435539a2910bb63a0e2af7d20b1/Source/ResourceMan.cpp#L51
https://github.com/OpenFodder/openfodder/blob/8504e7ab7d44d435539a2910bb63a0e2af7d20b1/Source/ResourceMan.cpp#L278-L296